autogen/variables.tf

```diff
@@ -112,11 +112,11 @@ variable "maintenance_start_time" {
 }
 
 variable "ip_range_pods" {
-  description = "The secondary ip range to use for pods"
+  description = "The subnet range name of the secondary ip range to use for pods"
```

Comment:
@emalloy this wording confuses me a bit - we're referring to the secondary IP subnet range name, yeah? The repetition in this description makes this a bit muddy, can we shorten this up a bit?